_data-prepper/log-analytics.md

```diff
@@ -0,0 +1,193 @@
+
+Here are all of the components for log analytics with FluentBit, Data Prepper, and OpenSearch:
+
+![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.png)
+
+In the application environment, you need to run FluentBit. FluentBit can be containerized through Kubernetes, Docker, or Amazon ECS. It can also run as an agent on EC2. Configure the [FluentBit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to export log data to Data Prepper. Then, deploy Data Prepper as an intermediate component and configure it to send the enriched log data to your OpenSearch cluster or Amazon OpenSearch Service domain. From there, use OpenSearch Dashboards to perform more intensive visualization and analysis.
+
+# Log analytics pipeline
+
+Log analytic pipelines in Data Prepper are extremely customizable. See the simple pipeline below.
+
+![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-ingestion-fluent-bit-data-prepper.png)
+
+# HTTP source
+
+The [HTTP Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md) accepts log data from FluentBit. This source accepts log data in a JSON array format and supports industry-standard encryption in the form of TLS/HTTPS and HTTP basic authentication.
+
+# Processor
+
+Data Prepper 1.2 release and above comes with a [Grok Processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/grok-processor/README.md). The Grok Processor is an invaluable tool to structure and extract important fields from your logs in order to make them more queryable.
+
+The Grok Processor comes with a wide variety of [default patterns](https://github.com/thekrakken/java-grok/blob/master/src/main/resources/patterns/patterns) that match against common log formats like apcahe logs or syslogs, but can easily accept any custom patterns that cater to your specific log format.
+
+There are a lot of complex Grok features that will not be discussed here, so please read the documentation if you are interested.
+
+# OpenSearch sink
+
+There is a generic sink that writes the data to OpenSearch as the destination. The [OpenSearch sink](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/README.md) has configuration options related to an OpenSearch cluster like endpoint, SSL/Username, index name, index template, index state management, etc.
+
+# Pipeline configuration
+
+## Example pipeline.yaml with SSL and basic authentication enabled
+
+Example `pipeline.yaml` with SSL and basic authentication enabled for the `http-source`:
+
+```yaml
+log-pipeline:
+  source:
+    http:
+      ssl: true
+      ssl_certificate_file: "/full/path/to/certfile.crt"
+      ssl_key_file: "/full/path/to/keyfile.key"
+      # The default port that will listen for incoming logs
```

Comment:
I think we can remove this comment the and port configuration. It is just the default.